load.Rmd

```diff
@@ -492,6 +492,7 @@ for(i in 1:nrow(cents)){
   cents$ebike_slc[i] <- sum(l$ebike_slc[j])
   cents$ebike_sic[i] <- sum(l$ebike_sic[j])
 
+  if(length(j) == 0) next
```

Comment:
The NANs were causing a problem for the geojson outputing, is it ok to just skip them @Robinlovelace ?